src/store/actions.js

```diff
@@ -7,9 +7,10 @@ export default {
   // Auth
   async facebookLogin({ commit, dispatch }, { status, authResponse }) {
     if (status === 'connected') {
-      const [data, err] = await post('/authenticate', {
-        fbToken: authResponse.accessToken,
-      })
+      const fbToken = authResponse.accessToken
+      const fcmToken = localStorage.getItem('fcmToken')
```

Comment:
Maybe store this in the vuex store? Also, if the user logs in before `getFcmToken` runs, we won't send the FCM token when it eventually arrives.